api/syncer.py

```diff
@@ -578,36 +597,43 @@ def expiring_access_notifications_owner() -> None:
         for owner in owners:
             non_owner_users = [user for user in users_per_group[group] if user.id != owner.id]
             if len(non_owner_users) > 0:
-                owner_expiring_groups_next[owner].users += non_owner_users
-                owner_expiring_groups_next[owner].groups.append(group)
+                owner_expiring_groups_next[owner].groups[group] = non_owner_users
 
     for owner in owner_expiring_groups_this:
         # If the owner has members with access expiring both this week and next week, only send one message
         if owner in owner_expiring_groups_next:
             notification_hook.access_expiring_owner(
                 owner=owner,
-                groups=owner_expiring_groups_this[owner].groups + owner_expiring_groups_next[owner].groups,
+                groups=None,
```

Comment:
For backwards compatibility (here and other notification plugin calls) can we send the previous values instead of None?